src/assets/content/dictionary/cache.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Cache
-definition:
-sources: []
-perspectives: []
+definition: Cache is a temporary storage space or memory that allows fast access to data, such as: Web browser cache; CPU cache.
```

Comment:
```suggestion
definition: 'Cache is a temporary storage space or memory that allows fast access to data, such as: Web browser cache; CPU cache.'
```
bug: Using a colon (`:`) in the markdown is breaking the build. I've made the suggestion to surround the whole string with quotes so that the compiler will know that the colon is content, not code. You can accept the change via the Github UI, or you can make an extra commit to this PR.